RELEASE.md

```diff
@@ -2,76 +2,33 @@
 
-The first step in a release is determining if the release is a feature release or a maintenance release. A maintenance release is a backward compatible bug fix or documentation change, while a feature release adds functionality in a backward compatible manner.
+## Versions and Stability
 
-#### Release Naming Conventions
+The SDK follows [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and therefore only major releases should break compatibility. Minor versions may include new functionality and patch versions address bugs or trivial changes (like documentation).
 
-This project follows [Semantic Versioning](https://semver.org/spec/v2.0.0.html), which specifies a version number as MAJOR.MINOR.PATCH with a feature release mapped to a MINOR version and a maintenance release mapped to a PATCH.
+If depending upon official packages from PyPI, a developer should feel comfortable specifying `planet == 2.*` unless depending on a specific feature introduced at a minor version, in which case `planet == 2.x.*` (where x is the minor version of the new feature) should suffice.
 
-The following are the release naming conventions:
+The default branch is always `main` and should be considered in-development but with tests and other build steps succeeding.
 
-1. Current Dev Version is obtained from `planet/__version__.py`
-    * For maintenance release, the format is `{MAJOR.MINOR.PATCH}dev`
-    * For feature release, the format is `{MAJOR.MINOR}dev`
-3. Release Version: Remove `dev` from Current Dev Version
-4. Next Dev Version: Bumped version of Release Version with `dev` added to the end.
-    * For maintenance release, bump PATCH
-    * For feature release, bump MINOR, do not specify PATCH
-5. Source branch:
-    * For maintenance release, source branch is `maint-{MAJOR.MINOR}`
-    * For feature release, source branch is the main branch, `main`
 
+## Release Workflow
 
-##### Maintenance Release Example:
-
-**IF** Current Dev Version ==  `1.1.1dev` **THEN**
-  * Release Version: `1.1.1`
-  * Next Dev Version: `1.1.2dev`
-  * Source Branch: `maint-1.1`
-
-
-##### Feature Release Example:
-
-**IF** Current Dev Version ==  `1.2dev` **THEN**
-  * Release Version: `1.2`
-  * Next Dev Version: `1.3dev`
-  * Source Branch: `main`
-
+1. Create a release branch off of `main` that bumps the SDK version number and updates the changelog:
+   * Bump SDK version number in `planet/__version__.py`
+2. Create a PR for the release branch and merge into `main`
+3. Create a new GitHub release:
+   * Set tag to release version
+   * Set target to `main`
```

Comment:
You can also tag in git and push the tag
e.g. 
* `git tag <release_version>`
* `git push <release_version>`
* Create a new github release with the target as the tag

Reply:
MR is updated with a workflow for tagging a new release via the `gh` CLI.  I tried tagging a release via `git tag` and while it does indeed create a new tag, it doesn't automatically create a new release entry.  With `gh release create` I can tag a new release and add release notes, which will create a new release entry in the releases UI in addition to the new tag.